src/Computer.js

```diff
@@ -0,0 +1,35 @@
+import { Random } from '@woowacourse/mission-utils'
+
+class Computer {
+    #solution;
+
+    seeSolution() {
+        return this.#solution
+    }
+
+    makeSolution() {
+        const computer = [];
+        while (computer.length < 3) {
+            const number = Random.pickNumberInRange(1, 9);
+            if (!computer.includes(number)) {
+                computer.push(number);
+            }
+        }
+        return this.#solution = computer
+    }
+
+    assessUserInput(input) {
+        let strike = 0;
+        let ball = 0;
+        const userInput = input.split("").map(item => Number(item));
```

Comment:
1. 이렇게도 됩니다.
```suggestion
        const userInput = input.split("").map(Number);
```
2. 배열이면.. s가 붙으면 좋지 않을까요?